### PR TITLE
Dynamic icons

### PR DESCRIPTION
**Pull request by cherimarie**
- add 'icon' column to providers table
- allow icon value to be chosen from limited set of available values (to be updated with real words when we get real icons) in rails admin form
- append the provider's icon value to the html element that holds it as a class, have css set the bakcground-image proerty of that element based on the class, with it defaulting to quhac logo
- no tests because I've got senioritis

Ability to set icons for a provider is functional. When we get the final icons from Sarah, they'll be switched in and it'll make more sense. The core functionality is in place now, with awesome fake icons and names by me. In the provider form, there's a field called "icon". It's a dropdown with options for 'a', 'b', 'c', and nothing. In the future, those options will be things like 'acupuncturist', 'MD'- basically all the icons we get from Sarah. You get to pick just one icon that best represents the provider. If you pick nothing, it'll get the default icon (right now, the hands clasped one). Then, in the providers list, on each provider card you'll see the appropriate icon. Hooray!

### CODE REVIEW

File: app/assets/stylesheets/results.css.scss [hunk 1]
```diff
@@ -14,12 +14,24 @@ p {margin: 0}
...
-  img {
-    max-width: 11rem;
+  .icon {
+    background-image: url('/assets/QuHac-logo.png');
+    background-repeat: no-repeat;
+    background-position: center;
     margin-top: -1.5rem;
-    @media (max-width: 992px) {max-width: 9rem;}
-    @media (max-width: 768px) {max-width: 7rem;}
+    @media (min-width: 993px) {min-width: 11rem; background-size: 11rem; height: 11rem; }
```
Comment (theutahkate) [inline app/assets/stylesheets/results.css.scss:22]: I don't think you need this as a media query. Setting those as the default styles on the class should be enough.

**Comment by theutahkate** [comment]
Just the one note, otherwise it looks great. Sweet icons.

File: app/assets/stylesheets/results.css.scss (hunk 1 above)
Comment (cherimarie) [inline app/assets/stylesheets/results.css.scss:22]: But they have to be different per media query size, right? I was trying to copy what you had before.

Comment (theutahkate) [inline app/assets/stylesheets/results.css.scss:22]: It is very possible I did exactly what you have here bc media queries turn me around sometimes. What I usually go for is a set of default styles, and then media queries for all the widths you want to not have those defaults.

**Pull request merged**